.markdownlint.json

```diff
@@ -16,6 +16,7 @@
   "MD035": {
     "style": "---"
   },
+  "MD034": false,
```

Comment:
Why are you disabling this rule? Should be consistent with https://github.com/acend/guidelines/blob/master/acend.json and across all projects